syntax = "proto3";

package latexocr;

// Interface exported by the server
service LatexOCR {
  // Generate the latex code for a given image filepath
  rpc GenerateLatex(LatexRequest) returns (LatexReply) {}
}

// The filepath for an image
message LatexRequest { string image_path = 1; }

// The latex code returned by the model
message LatexReply { string latex = 1; }